### PR TITLE
[DASH-2413] Migrate to webpack2

### PR DESCRIPTION
**Pull request by patrykkopycinski**

### CODE REVIEW

File: src/apps/Profile/ProfileBillingPlanCancel.js [hunk 1]
```diff
@@ -52,7 +52,9 @@ const ProfileBillingPlanCancel = Radium(React.createClass({
           There was an error sending your request.
         </div>
         <div>
-          Sorry about that. Please write us at <a href={`mailto:${SYNCANO_BILLING_EMAIL}`}>{SYNCANO_BILLING_EMAIL}</a>.
+          {'Sorry about that. Please write us at '}
```
Comment (tomsonpl) [inline src/apps/Profile/ProfileBillingPlanCancel.js]: if you are going to make another commit - change this please, 'write us' is incorrect. I suggest contact us

File: README.md [hunk 2]
```diff
@@ -143,13 +143,13 @@ To install dependencies just do:
 
 ```sh
 $ cd syncano-dashboard/
-$ npm install
+$ yarn install
```
Comment (wpasinski) [inline README.md]: or even just `$ yarn`.
